### DISCUSSION TITLE
Deploy directory

### DISCUSSION THREAD

**Discussion by pabloto** [codefori/vscode-ibmi#1113] [Q&A]
Hi, 

Could the deploy directory be an env var?

So I can set it different for users or branch.

Many thanks

**Comment by worksofliam** [comment]
Interesting idea. Let me see how this would work fundamentally.

**Comment by pabloto** [comment]
Yesterday I encountered this problem.
I’m working on a change for a new project relative to my repository, in a
new branch, but I have to fix a bug, so a checked out to main branch,
correct the bug, and probably I make a mistake, (but really not sure) I
deploy with options working change after commit my changes. So the source
present in my deploy directory isn’t the source I have committed, but an
old source ( two commit ago), this source is the source in my new branch,
but I didn’t change it in the new branch.
So if exist an env var to define which is the directory to deploy for every
branch and for every users, I can put this files into .gitignore and
everyone wants to clone this repo, and deploy from his properly branch ,
have to set the deploy directory and I avoid it he problem then I
encountered.

Manny thanks

Il giorno ven 3 mar 2023 alle 22:51 barry ***@***.***> ha
scritto:

> Interesting idea. Let me see how this would work fundamentally.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/orgs/halcyon-tech/discussions/1113#discussioncomment-5197359>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ACKE4EIMHHJP6V25QOMWD73W2JRVLANCNFSM6AAAAAAVO6REBM>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***
> com>
>
-- 
Paolo Salvatore
